sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/AzureBlobStorageBuilder.java

```diff
@@ -91,7 +91,11 @@ public AzureBlobStorageImpl build() {
         if (this.url != null) {
             client.setUrl(this.url);
         }
-        client.setVersion(this.version);
+        if (this.version != null) {
+            client.setVersion(this.version);
+        } else {
+            client.setVersion("2019-02-02");
```

Comment:
We should never let version be null

Reply:
Codegen artifact, our client builders should never get into this state.